css/notes.css

```diff
@@ -9,7 +9,6 @@
 }
 
 #app-navigation #note-add span {
-    display: none;
```

Comment:
@jancborchardt what do you think about this?
We get the demand to show the text every once in a while.